nuspec/chocolatey/GitReleaseManager.Portable.nuspec

```diff
@@ -10,8 +10,8 @@
     <projectUrl>https://github.com/GitTools/GitReleaseManager</projectUrl>
     <iconUrl>https://cdn.jsdelivr.net/gh/GitTools/GitReleaseManager@9c547452a10afaf83fce1b5833f4762487b017b7/Icons/package_icon.svg</iconUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Tool for creating and exporting releases for software applications hosted on GitHub</description>
-    <summary>Tool for creating and exporting releases for software applications hosted on GitHub</summary>
+    <description>Tool for creating and exporting releases for software applications</description>

```

Comment:
instead of removing `hosted on GitHub`, maybe change it to `hosted on a remote provider`?

Reply:
Done